packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,35 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+
+/**
+ * Runs an operation and saves the return value to a pre-defined key in state
+ * @public
+ * @function
+ * @example
+ * as('lookup_table.option_set', get('fixture/?fixture_type=option_set_mapping'));
+ * @param {string} key - The key used to store the return value in the state object
```

Comment:
Note for me: Perhaps as well as the string, we can accept an object which maps stuff. So `"key"` and `{ data: "key" }` would be equivalent.

This would bypass the problems of a) only `state.data` gets mapped and `state.data` is kind of arbitrary, and b) what if I want to map multiple state keys, like http data AND response?

Please don't implement this yet, it might be too complicated

Reply:
Yes, too complicated. Maybe we'll add it later if there's a demand.